lldb/source/Plugins/ScriptInterpreter/Python/Interfaces/CMakeLists.txt

```diff
@@ -19,6 +19,12 @@ if (LLDB_ENABLE_LIBEDIT)
   list(APPEND LLDB_LIBEDIT_LIBS LibEdit::LibEdit)
 endif()
 
+set_property(GLOBAL PROPERTY LLDB_EXTRA_SCRIPT_PLUGINS
+  lldbPluginOperatingSystemPythonInterface
+  lldbPluginScriptedProcessPythonInterface
+  lldbPluginScriptedThreadPlanPythonInterface
```

Comment:
@bulbazord aren't we making 3 separate libraries here ?

Reply:
No, you're just adding some elements to the list `LLDB_EXTRA_SCRIPT_PLUGINS`. Right below this is `add_lldb_library` which creates one library. It will contain the 3 cpp files that implement these plugin interfaces.